packages/datadog_flutter_plugin/README.md

```diff
@@ -1,39 +1,38 @@
-<p align="center">
-  <img src="dd_logo.png" width="200">
-</p>
```

Comment:
Also, I'm sad to remove the logo. It looks really good on the pub.dev site. Maybe something to look into for the future is a way to mark sections of the file to be excluded from the docs site?